src/main/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiService.scala

```diff
@@ -228,12 +230,15 @@ trait MethodConfigApiService extends HttpService with PerRequestCreator {
     new ApiResponse(code = 409, message = "Destination method configuration by that name already exists"),
     new ApiResponse(code = 500, message = "Rawls Internal Error")
   ))
-  def copyMethodRepoConfigurationRoute = cookie("iPlanetDirectoryPro") { securityTokenCookie =>
-    path("methodconfigs" / "copyFromMethodRepo") {
-      post {
-        entity(as[MethodRepoConfigurationQuery]) { query =>
-          requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
-            WorkspaceService.CopyMethodConfigurationFromMethodRepo(query, securityTokenCookie))
+  def copyMethodRepoConfigurationRoute = usernameFromCookie() { userId =>
+    // TODO: reads the cookie twice!
```

Comment:
if usernameFromCookie returned an object that contains the user id and security token we can read it once, also it would give us flex to add other things if we need to. maybe call it userInfoFromCookie?